scripts/zones/Aydeewa_Subterrane/mobs/Pandemonium_Warden.lua

```diff
@@ -21,9 +44,41 @@ end;
 function onMobSpawn(mob)
     -- Make sure model is reset back to start
     mob:setModelId(1839);
+    
+    -- Prevent death and hide HP until final phase
+    mob:setUnkillable(true);
+    mob:hideHP(true);
 
     -- Two hours to forced depop
     mob:setLocalVar("PWardenDespawnTime", os.time(t) + 7200);
+    mob:setLocalVar("phase", 1);
+    mob:setLocalVar("astralFlow", 1);
+end;
+
+-----------------------------------
+-- onMobDisengage Action
+-----------------------------------
+
+function onMobDisengage(mob)
+    -- Make sure model is reset back to start
+	mob:setModelId(1839);
+    
+    -- Prevent death and hide HP until final phase
+    mob:setUnkillable(true);
+	mob:hideHP(true);
```

Comment:
No idea how that happened, changed my settings a long time ago. Whatever, fixing.